app/assets/stylesheets/components/shared/c-card.scss

```diff
@@ -1,14 +1,18 @@
 
 .c-card {
+  display: flex;
   position: relative;
-  margin: 0 0 20px;
+  flex-direction: column;
+  height: $card-height;
```

Comment:
By design, cards can not be longer than that height. In case the title or summary are long ones, the content will override picture height, not card one. Anyway, in the BO there's is also a character limitation for those fields because we knew there could be problems. 

We can not allow having cards with different heights based on its content, that's why designers told me long time ago.